main.py

```diff
@@ -1,10 +1,49 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    # We read the data from the players.json file
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+    # exit the context manager after reading data from the file
+    print(players_data)
+
+    for nickname, player_data in players_data.items():
+        # First we get or create a race
+        race_data = player_data["race"]
+        race, _ = Race.objects.get_or_create(
+            name=race_data["name"],
+            defaults={"description": race_data.get("description", "")}
+        )
+        # Get or create a guild
+        guild_data = player_data.get("guild", None)
```

Comment:
There is no need to explicitly define `None` in `get`, it is a default option. Rewrite this line
```suggestion
        guild_data = player_data.get("guild")
```